azerothcore/Keira3
# perf(on-push): migrate most components of of features/*

- Part of https://github.com/azerothcore/Keira3/issues/2602

In this PR `121` out of the `131` Components of the application have been migrated to `OnPush` (all of those who did not break any automated tests). The remaining `10` will be migrated later.

How to test:

1) Clone Keira3: `git clone git@github.com:azerothcore/Keira3.git && cd Keira3`
2) Install dependencies: `npm i`
3) Checkout this branch: `git checkout onpush-migration-phase-1`
4) Start the application `npm start`
5) Play with the application and see if there are new bugs
